sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Client/CryptographyClient.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Client
+{
+    using Azure.Core;
+    using Azure.Core.Pipeline;
+    using Azure.Security.KeyVault.Cryptography.Base;
+    using Azure.Security.KeyVault.Cryptography.CryptoProvider;
+    using Azure.Security.KeyVault.Cryptography.Interface;
+    using Azure.Security.KeyVault.Cryptography.Utilities;
+    using Azure.Security.KeyVault.Keys;
+    using System;
+
+    /// <summary>
+    /// The Cryptography Client 
+    /// </summary>
+    public class CryptographyClient
+    {
+        #region const
+        const string DEFAULT_KV_SCOPE_URI = @"https://vault.azure.net/.default";
+        #endregion
+
+        #region fields
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public TokenCredential Credentials { get; }
+        /// <summary>
+        /// Cryptography Client Options
+        /// </summary>
+        public CryptographyClientOptions CryptoClientOptions { get; }
```

Comment:
same comment as above, we shouldn't expose this

Reply:
are you saying, we should not expose it as public?
because we will need it to be at least internal
Are we allowed to have have internal properties on public types in our new v2 wave of SDKs?